### PR TITLE
Updated MD5 to Sha1

### PR DESCRIPTION
**Pull request by baderbuddy**
Also updated version number.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "hash-for-dep",
-  "version": "1.0.2",
+  "version": "1.0.3",
```
Comment (kellyselden) [inline package.json]: This would probably be a major version change. Nevertheless, the version probably shouldn't be touched in this PR.

Comment (baderbuddy) [inline package.json]: Thanks for the note, I undid the version change.

Comment (stefanpenner) [inline package.json]: hmm, ya im unsure if this is major or minor. I guess we never really thought about that. I'll think on it...

**Comment by stefanpenner** [comment]
Im a-bit busy, but I can likely release tonight.

**Pull request merged**